### DISCUSSION TITLE
The Doks showcase section is updated!

### DISCUSSION THREAD

**Discussion by h-enk** [thuliteio/doks#557] [Showcase]
![Snag_2036ac9e](https://user-images.githubusercontent.com/3902872/141995840-2b275dc2-30ff-4e08-a378-4dc8d83735af.png)

Now, you can filter for type and functionalities used. Check out what others have build with Doks!

https://getdoks.org/showcase/

**Comment by bwnyasse** [comment]
@h-enk  Thank you ! How do we enable it in the site. I would like to implement a kind of showcase page in our site. Thank you

**Comment by h-enk** [comment]
@bwnyasse, not something to be incorporated into Doks. 

If you'd like to implement a Showcase section, have a look at how I did it for getdoks.org: https://github.com/h-enk/getdoks.org/tree/master/layouts/showcase + https://github.com/h-enk/getdoks.org/tree/master/content/showcase
